cypress/integration/login.js

```diff
@@ -1,27 +1,103 @@
+import LoginPage from '../objects/loginPage'
+import TimeLoginPage from '../objects/timeLoginPage'
+
+const loginPage = new LoginPage()
+const timeLoginPage = new TimeLoginPage()
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
         
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+        loginPage.visit()
+        loginPage.getValidationError().should('not.visible')
+        loginPage.getSubmit().click()
+        loginPage.getValidationError().should('be.visible')
     })
 
     it('Should be able to login with role User and should see today date', function () {
-
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Ugne Lekstutiene"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
-        cy.get('[type="submit"]').click();
-        cy.url().should('include', '/time-logging');        
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.calendar--today').contains(new Date().getDate());
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Ugne Lekstutiene');
-        cy.get('.main-nav').find('li').should('have.length', 1);        
+        
+        loginPage.visit()
+        loginPage.getUserForm().click({force:true})
+        loginPage.getUserOption("Ugne Lekstutiene").click()
+        loginPage.getLoginForm().click({force:true})
+        loginPage.getLabel('User').click()
+        loginPage.getSubmit().click()
+        timeLoginPage.getURL().should('include', '/time-logging')
+        timeLoginPage.getPageTitle('Timesheets')
+        timeLoginPage.getCalendarVisable().should('be.visible')
+        timeLoginPage.getCalendarToday().contains(new Date().getDate())
+        timeLoginPage.getTimeEntryForm('be.visible')
+        timeLoginPage.getUserInfoTitle('Ugne Lekstutiene')
+        timeLoginPage.getMainLength('1')  
+        timeLoginPage.getTimeLoggingColored('rgb(64, 76, 237)')

```

Comment:
There's no point in inserting the RGB color parameter here since this function does not even take any parameters :) You already assert the blue color in the function definition